content/api/cypress-api/cookies.md

```diff
@@ -153,8 +153,9 @@ single test.
 
 <Alert type="info">
 
-A great place to put this configuration is in your `cypress/support/index.js`
-file, since it is loaded before any test files are evaluated.
+A great place to put this configuration is in the
```

Comment:
It almost seems like this content should be in a partial 😂

Reply:
I can dig it. It is reoccurring!